Vaadin-server depends on old jsoup version, current versions throws exception

Only use GitHub issues for bugs and feature requests.   
For general support from the community, see https://vaadin.com/forum

NOTE: Issues for Vaadin Flow (Vaadin v10 Java Framework) should go to https://github.com/vaadin/flow

To make sure we can quickly validate and fix the bug, please provide as much as possible of the following information:
- Vaadin Framework version (e.g. 8.10.3+)
- Description of the bug
`javax.servlet.ServletException: com.vaadin.server.ServiceException: java.lang.NoSuchMethodError: 'void org.jsoup.nodes.DataNode.<init>(java.lang.String, java.lang.String)'
	at com.vaadin.server.VaadinServlet.service(VaadinServlet.java:450)`
- Minimal reproducible example
upgrade jsoup 1.12.2 -> 1.13.1, no such
- Expected behavior
To not throw exception
- Actual behavior
Throws exception

Would be good if next release after 8.11.0.alpha1 upgraded jsoup version or at least didn't break with current version 1.13.1

http://javadox.com/org.jsoup/jsoup/1.12.2/org/jsoup/nodes/DataNode.html

DataNode​(String data,String baseUri)	
Deprecated.
use DataNode(String) instead

So minor change using DataNode(String) constructor available in both versions and getting away from deprecated version.

Was another error

`Caused by: java.lang.NoSuchMethodError: 'void org.jsoup.nodes.DocumentType.<init>(java.lang.String, java.lang.String, java.lang.String, java.lang.String)'
	at com.vaadin.server.BootstrapHandler.setupStandaloneDocument(BootstrapHandler.java:514)
	at com.vaadin.server.BootstrapHandler.getBootstrapHtml(BootstrapHandler.java:459)
	at com.vaadin.server.BootstrapHandler.synchronizedHandleRequest(BootstrapHandler.java:428)
	at com.vaadin.server.SynchronizedRequestHandler.handleRequest(SynchronizedRequestHandler.java:40)
`
Similar with another 
deprecated constructor http://javadox.com/org.jsoup/jsoup/1.12.2/org/jsoup/nodes/DocumentType.html

--- a/server/src/main/java/com/vaadin/server/BootstrapHandler.java
+++ b/server/src/main/java/com/vaadin/server/BootstrapHandler.java
@@ -510,8 +510,7 @@ public abstract class BootstrapHandler extends SynchronizedRequestHandler {
 
         Document document = response.getDocument();
 
-        DocumentType doctype = new DocumentType("html", "", "",
-                document.baseUri());
+        DocumentType doctype = new DocumentType("html", "", "");
         document.child(0).before(doctype);
 
         Element head = document.head();

